File: Poin B/Python/SoccerClub.py
from mimetypes import init


class SoccerClub:
    # attribute
    __Nama = ""
    __Nation = ""
    __Year = ""
    __Players = ""

    def __init__(self, Namac = "", Nationc = "", Yearc = "", Playersc = ""):
        self.__Nama = Namac
        self.__Nation = Nationc
        self.__Year = Yearc
        self.__Players = Playersc

    def setNama(self, Namac):
        self.__Nama = Namac

    def getNama(self):
        return self.__Nama

    def setNation(self, Nationc):
        self.__Nation = Nationc

    def getNation(self):
        return self.__Nation

    def setYear(self, Yearc):
        self.__Year = Yearc

    def getYear(self):
        return self.__Year

    def setPlayers(self, Playersc):
        self.__Players = Playersc

    def getPlayers(self):
        return self.__Players

    class Trophy:
        __NamaKejuaraan = ""
        __Tahun = ""
        __Tipe = ""

        def __init__(self, NamaKejuaraanc = "", Tahunc=0, Tipec=""):
            self.__NamaKejuaraan = NamaKejuaraanc
            self.__Tahun = Tahunc
            self.__Tipe = Tipec

        def setNamaKejuaraan(self, NamaKejuaraanc):
            self.__NamaKejuaraan = NamaKejuaraanc

        def getNamaKejuaraan(self):
            return self.__NamaKejuaraan

        def setTahun(self, Tahunc):
            self.__Tahun = Tahunc

        def getTahun(self):
            return self.__Tahun

        def setTipe(self, Tipec):
            self.__Tipe = Tipec

        def getTipe(self):
            return self.__Tipe

        def OutputInner(self):
            print("Nama Kejuaraan   : " + self.__NamaKejuaraan)
            print("Tahun            : " + self.__Tahun)
            print("Tipe             : " + self.__Tipe)

    __Tp = Trophy()

    def setTrophy(self, NamaKejuaraan, Tahun, Tipe):
        self.__Tp.setNamaKejuaraan(NamaKejuaraan)
        self.__Tp.setTipe(Tipe)
        self.__Tp.setTahun(Tahun)

    def getTrophy(self):
        return self.__Tp

        # print method

    def Output(self):
        print("Team Name    : " + self.getNama())
        print("Country      : " + self.getNation())
        print("Established  : " + self.getYear())
        print("Players      :")
        for i in self.getPlayers():
            print("               - " + i)

File: Poin B/Java/SoccerClub.java
import java.util.*;

public class SoccerClub{
    
    private String nama;
    private String nation;
    private String year;
    private String[] players;
    private Trophy tp;
    
    public SoccerClub(String namac, String nationc, String yearc, String[] playersc){
        this.nama = namac;
        this.nation = nationc;
        this.year = yearc;
        this.players = playersc;
    }

    public SoccerClub(){

    }

    public String getName(){
        return this.nama;
    }

    public void setName(String namac){
        this.nama = namac;
    }

    public String getNation(){
        return this.nation;
    }

    public void setNation(String nationc){
        this.nation = nationc;
    }

    public String getYear(){
        return this.year;
    }

    public void setYear(String yearc){
        this.year = yearc;
    }

    public String[] getPlayers(){
        return this.players;
    }

    public void setPlayers(String[] playersc){
        this.players = playersc;
    }

    
    public void output(){
        System.out.println("Name Team   : "+ this.getName());
        System.out.println("Country     : "+ this.getNation());
        System.out.println("Established : "+ this.getYear());
        System.out.println("Players     : ");

        String[] arrPlayers = new String[this.players.length];
        arrPlayers = this.players;
        int i;
        for(i = 0; i< arrPlayers.length; i++){
            System.out.println("              - "+ arrPlayers[i]);
        }

    }

    public void setTrophy(String namaKejuaraanc, String tahunc, String typec){
        tp = new Trophy();
        tp.setNamaKejuaraan(namaKejuaraanc);
        tp.setTahun(tahunc);
        tp.setType(typec);
    }

    public Trophy getTrophy(){
        return tp;
    }

    public class Trophy{
        private String namaiKejuaraan;
        private String tahun;
        private String type;

        public Trophy(String namaKejuaraanc, String tahunc, String typec){
            this.namaiKejuaraan = namaKejuaraanc;
            this.tahun = tahunc;
            this.type = typec;

        }

        public Trophy(){

        }

        public String getNamaKejuaraan(){
            return this.namaiKejuaraan;
        }

        public String getTahun(){
            return this.tahun;
        }

        public String getType(){
            return this.type;
        }

        public void setNamaKejuaraan(String namaKejuaraanc){
            this.namaiKejuaraan = namaKejuaraanc;
        }

        public void setTahun(String tahunc){
            this.tahun = tahunc;
        }

        public void setType(String typec){
            this.type = typec;
        }

        public void outputInner(){
            System.out.println("Last Trophy : "+ this.getNamaKejuaraan());
            System.out.println("Year        : "+ this.getTahun());
            System.out.println("Type        : "+ this.getType());
    
        }

    }
    




}